pages/linux/tlp-stat.md

```diff
@@ -8,10 +8,30 @@
 
 `sudo tlp-stat`
 
-- Show battery information:
+- Show information about various devices:
 
-`sudo tlp-stat -b`
+`sudo tlp-stat --{{battery|disk|processor|graphics|pcie|rfkill|usb}}`
+
+- Show verbose information about devices that support verbosity:
+
+`sudo tlp-stat --verbose --{{battery|processor|pcie|usb}}`
 
 - Show configuration:
 
-`sudo tlp-stat -c`
+`sudo tlp-stat {{-c|--config}}`
+
+- Monitor power supply `udev` events:
+
+`sudo tlp-stat {{-P|--pev}}`
+
+- Show power supply diagonistics:
+
+`sudo tlp-stat --psup`
+
+- Show temperatures and fan speed:
```

Comment:
```suggestion
- Show [temp]eratures and fan speed:
```

Reply:
Should this be just `[t]` or `[temp]`?

Reply:
IMO when there is a short option and a **abbreviated** short option the priority is the mnemonic for the abbreviated long option because it adds more value, but this is just a personal opinion, especially since mnemonics for long options are not common.